tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+isolated_build = true
+env_list = lint,py{310,311,312,313},test
```

Comment:
does that work if I run tox locally (where I don't have all the different python versions installed) ?